lib/reservation_monitor.py

```diff
@@ -101,24 +101,38 @@ def _check_flight_fares(self) -> None:
             try:
                 fare_checker.check_flight_price(flight)
                 self.notification_handler.healthchecks_success(
-                    f"Successful fare check, confirmation number={flight.confirmation_number}"
```

Comment:
Is there a reason all of these were changed? It is just for healthchecks and the one-liner doesn't seem like it would be that hard to read--especially if this is only to make sure the fare checker is running.

Reply:
Slight appearance change from this:
![IMG_63771F0694C4-1](https://github.com/jdholtz/auto-southwest-check-in/assets/26825098/d74e9e4b-ae19-4cf1-93cf-044efe6db3ca)

to look like:
'
Successful fare check,
confirmation = 24K3XS
'

Reply:
Gotcha, yeah that looks better. Thanks

Reply:
Southwest uses 'confirmation #: 3KXSSA'

Should I make a change to be:
"
Successful fare check,
confirmation #: 3KXSSA
"

Reply:
`confirmation number = ...` is good. It doesn't need to be 100% consistent with Southwest's website and I think this is easier to understand from a glance.